14/umbraco-cms/implementation/unit-testing.md

```diff
@@ -159,23 +159,21 @@ public class PageSurfaceControllerTests
 `ServiceContext.CreatePartial()` has optional parameters, and by naming them you only need to mock the dependencies that you need, for example: `ServiceContext.CreatePartial(contentService: Mock.Of<IContentService>());`
 {% endhint %}
 
-## Testing an UmbracoApiController
+## Testing an ApiController
 
 See [Reference documentation on UmbracoApiControllers](../reference/routing/umbraco-api-controllers/README.md#locally-declared-controller).
```

Comment:
Should we change UmbracoApiControllers to ApiControllers here? 🤔

Reply:
@sofietoft Not sure. Maybe it's better to change it to Controller?

Reply:
Yes, sure! I think Controller is good 💪 

Did you see my comments [above as well](https://github.com/umbraco/UmbracoDocs/pull/6524#pullrequestreview-2356241504)? 🤞